pages/tasks/index.js

```diff
@@ -3,23 +3,56 @@ import Head from 'next/head'
 import { withProtected } from '../../hooks/route'
 import { getAllTasks } from '../../lib/tasks'
 import { useTranslation } from 'react-i18next'
+import SearchBar from '../../components/SearchBar'
+import { MdGroup } from 'react-icons/md'
+import { AiOutlineLike } from 'react-icons/ai'
+import { Button, Image, Text } from '@nextui-org/react'
+import Filter from '../../components/FilterTasks/index'
+import { useTheme } from 'next-themes'
 
 const IssueCard = ({ issue }) => {
+  const { t } = useTranslation()
+  const { theme } = useTheme()
+  const isLight = theme === 'light'
   return (
-    <div className="m-2 flex flex-col justify-between rounded-lg border border-gray-200 bg-gray-100 p-4 shadow-lg">
-      <div className="mb-4">
-        <h3 className="text-lg text-indigo-500">{issue.title}</h3>
+    <div
+      className={`flex md:flex-row flex-col items-center justify-center gap-2 rounded-lg p-4 shadow-lg ${
+        isLight ? 'bg-gray-200 bg-opacity-75' : 'bg-black-200 bg-opacity-75'
+      }`}
+    >
+      <div className="flex h-[40px] md:h-full w-[40px] md:w-[80px] items-center justify-center md:rounded-[20px] rounded-[10px] bg-[#99e24d]">
+        <MdGroup size={70} color="white" />
       </div>
-      <div className="text-sm text-gray-700">
-        <p>
-          <strong>Board:</strong> {issue.project_name}
-        </p>
-        <p>
-          <strong>Created At:</strong> {new Date(issue.createdAt).toLocaleDateString()}
-        </p>
-        <p>
-          <strong>Updated At:</strong> {new Date(issue.updatedAt).toLocaleDateString()}
-        </p>
+      <div className="mb-4 flex w-full flex-col gap-3">
+        <div className="flex w-full items-center justify-between">
+          <span className="md:text-[24px] text-[18px] text-[#99e24d]">{issue.title}</span>
+          <button className="text-white mr-2 md:mr-4 rounded bg-[#99e24d] bg-opacity-30 px-2 py-1 md:text-sm text-[10px] hover:bg-[#649e26] focus:outline-none focus:ring-2 focus:ring-[#99e24d]">
+            Apply
+          </button>
+        </div>
+        <div className="flex w-full">
+          <p className="text-[12px] md:text-[16px]">
+            Our practical learning platform, located at build.w3d.community, offers courses focused
```

Comment:
this text shouldn't be hardcoded, this should be a dynamic text from issues

Reply:
@nomadbitcoin, I have already adjusted this information to come from the body field of our backend (seed-data.json)